Cargo.toml

```
@@ -12,7 +12,7 @@ license = "Apache-2.0"
 fam-wrappers = ["vmm-sys-util"]
 
 [dependencies]
-vmm-sys-util = { version = "0.11.0", optional = true }
+vmm-sys-util = { version = "0.11.0", optional = true, features = ["with-serde"] }
 
-versionize = { version = "0.1.6" }
-versionize_derive = { version = "0.1.3" }
+serde = { version = "1.0.136", features = ["derive"] }
+serde_with = "3.4.0"
```

nit:
```suggestion
serde_with = {version = "3.4.0", default-features = false, features = ["macros"]}
```
brings down the dependencies a bit. Or maybe we can use https://crates.io/crates/serde-big-array, to get the dependency count down even more? I'm just a bit worried that we pull lots of things we don't use into our Cargo.lock, which will increase the risk of us getting hit with cargo audit warnings (e.g. we pull in "bumpalo" again, which we had triggered an audit warning for us in the past)

---

Done!